ext/reflection/php_reflection.stub.php

```diff
@@ -764,6 +764,8 @@ class ReflectionAttribute implements Reflector
     /** @cvalue REFLECTION_ATTRIBUTE_IS_INSTANCEOF */
     public const int IS_INSTANCEOF = UNKNOWN;
 
+    public string $name;
```

Comment:
Should be readonly?

Reply:
Perhaps, although the others aren't. The readonly-ness is handled manually by `_reflection_write_property` and it throws a ReflectionException. So for consistency I went this route.
The readonly modifier makes it throw an Error, so changing the behaviour of the others would be a small BC break.

Reply:
I also think this should be possible because changing the property makes zero sense, and makes no difference. Some people may naively think that the name property has an effect on the attribute itself.

TBH we also made a few properties readonly in the past, so I think it's doable now as well, unless some very popular library (e.g. BetterReflection) redefine them.

Reply:
OK, I'll check BetterReflection some time later and if that's okay I'll drop the special handling and let these use `readonly`

Reply:
Here is the patch to remove the special handling: https://gist.github.com/nielsdos/e69d6b826fecf60a7c3fedce3f83ff7f

However, we can't apply that patch. BetterReflection relies on the behaviour (e.g. https://github.com/Roave/BetterReflection/commit/3eedf48bd38aaf716eb262e7c2faea7679e51337 but there are more cases, also exception classes in `catch` would be required to change) and that patch therefore causes 562 test failures.